### PR TITLE
Fix broken links in pull_request_template

### PR DESCRIPTION
**Pull request by Bharat123rox**
## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## What is the current behavior?

Broken links in pull_request_template.md

## What is the new behavior?

Fixed links in pull_request_template.md

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -32,12 +32,12 @@ What kind of change does this PR introduce?
...
-- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
+- [ ] Tested code with current [supported SDKs](../README.md#uno-features)
+- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
+- [ ] [Unit Tests and/or UI Tests](../doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
+- [ ] [Wasm UI Tests](../doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
 - [ ] Contains **NO** breaking changes
-- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
+- [ ] Updated the [Release Notes](../doc/ReleaseNotes)
```
Comment (carldebilly) [inline .github/pull_request_template.md]: Not sure relative links will work there.

Comment (ghuntley) [reply .github/pull_request_template.md]: Relative works in preview mode but I think the PR should be re-worked to use absolute as the links are able to be copied and pasted into a new window during authoring - which is not possible with relative.

Comment (Bharat123rox) [reply .github/pull_request_template.md]: Replaced

**Comment by ghuntley** [comment]
Thanks @Bharat123rox for incorporating the feedback and your help to make Uno better. Head on over to https://platform.uno/uno-is-joining-hacktoberfest-2019/ and follow the instructions on that website and we'll send you some swag.

**Comment by ghuntley** [comment]
@all-contributors please add @Bharat123rox for docs

**Pull request merged**